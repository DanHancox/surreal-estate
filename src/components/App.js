import "../styles/app.css";
import NavBar from "./NavBar";
import React from "react";

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <NavBar />
      </header>
    </div>
  );
}

export default App;
