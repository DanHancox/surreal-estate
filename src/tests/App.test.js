import { render, screen } from "@testing-library/react";
import App from "../components/App.js";

describe("App", () => {
  it("renders App", () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });
});

xtest("renders learn react link", () => {
  render(<App />);
  const linkElement = screen.getByText(/Surreal Estate/i);
  expect(linkElement).toBeInTheDocument();
});
