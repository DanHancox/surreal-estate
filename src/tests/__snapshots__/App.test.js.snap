// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders App 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <header
      class="App-header"
    >
      <div>
        <nav
          class="navBar"
        >
          <img
            alt="houselogo"
            class="logoPic"
            src="https://mcrcodes.s3.eu-west-2.amazonaws.com/course/surreal-estate/logo.png"
          />
          <ul
            class="navbar-links"
          >
            <li
              class="navbar-links-item"
            >
              <a
                href="View Properties"
              >
                View Properties
              </a>
            </li>
            <li
              class="navbar-links-item"
            >
              <a
                href="add-property"
              >
                Add a Property
              </a>
            </li>
          </ul>
        </nav>
      </div>
    </header>
  </div>
</DocumentFragment>
`;
